RAM_Adapter/RAMAdapter.cs

```diff
@@ -52,6 +52,8 @@ public RAMAdapter(string filePath = "", bool active = false)
                 SetupDependencies();
                 SetupComparers();
 
+                m_AdapterSettings.UseAdapterId = false;
```

Comment:
As an alternative to this, could instead change the caching toggle, to turn of the caching system being used.

This should bring the toolkit back to work to how it was before the caching system introduction.

Will leave the final call to you.
```suggestion
                m_AdapterSettings.CacheCRUDobjects = false;
```

Reply:
With "m_AdapterSettings.UseAdapterId = false;" some of the panels aren't being pushed properly (1. Panel  has a section property with no AdapterID, so the deck could not be assigned in RAM.), but "m_AdapterSettings.UseAdapterId = false;" results in successful pushes and updates on my end so I say we go with that.